src/pixi/InteractionData.js

```diff
@@ -43,10 +43,10 @@ PIXI.InteractionData = function()
  * @param [point] {Point} A Point object in which to store the value, optional (otherwise will create a new point)
  * @return {Point} A point containing the coordinates of the InteractionData position relative to the DisplayObject
  */
-PIXI.InteractionData.prototype.getLocalPosition = function(displayObject, point)
+PIXI.InteractionData.prototype.getLocalPosition = function(displayObject, point, globalPos)
 {
     var worldTransform = displayObject.worldTransform;
-    var global = this.global;
+    var global = globalPos ? globalPos : this.global;
```

Comment:
you should check for optional params like so:

```javascript
var global = (globalPos === 'undefined') ? this.global : globalPos;
```

The logic fails if globalPos is passed, but evaluates as false.